float32 value
---
bool result
